### PR TITLE
Add mautrix-wsproxy role (to enable mautrix-imessage/Beeper's Brooklyn) from hanthor

### PR DESCRIPTION
**Pull request by shreyasajj**

### CODE REVIEW

File: group_vars/matrix_servers [hunk 1]
```diff
@@ -607,6 +607,48 @@ matrix_mautrix_whatsapp_database_password: "{{ '%s' | format(matrix_homeserver_g
...
+
+matrix_mautrix_wsproxy_homeserver_address: "{{ 'http://matrix-synapse:8008' if matrix_synapse_enabled else '' }}"
+
+matrix_mautrix_wsproxy_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'wsproxy.hs.token') | to_uuid }}"
+
+matrix_mautrix_wsproxy_appservice_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'wsproxy.as.token') | to_uuid }}"
+
+matrix_mautrix_imessage_homeserver_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'imessage.hs.token') | to_uuid }}"
+
+matrix_mautrix_imessage_appservice_token: "{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'imessage.as.token') | to_uuid }}"
```
Comment (spantaleev) [inline group_vars/matrix_servers:638]: All of these should be switched from `matrix_synapse_macaroon_secret_key` to `matrix_homeserver_generic_secret_key`. See 965890bf757ab0d9

**Comment by josh900** [comment]
Is this ok to use?

**Comment by jswetzen** [comment]
@shreyasajj are you still working on this? This is the last piece missing to make matrix my one-stop messaging shop. The other Android SMS bridge projects (as this doubles as) seem unmaintained or moro cumbersome.

**Comment by jswetzen** [comment]
@shreyasajj @hanthor @josh900 and anyone else who is interested in this: I've merged with master and fixed the linting issues [in my repo](https://github.com/jswetzen/matrix-docker-ansible-deploy/tree/mautrix-wsproxy) and might do a new PR to superseed this. It seems though that it really needs the [syncproxy](https://github.com/mautrix/syncproxy) to be useful for Android SMS so I'm looking into adding that as well.

**Comment by hanthor** [comment]
Definitely make a new PR. I created this with pretty limited knowledge of the code and Matrix API.

**Comment by shreyasajj** [comment]
I kind of gave up as I have a unique setup and I think it mostly worked, but no one would test it. Also kind of unsatified with having to create a whole new dns entry instead of creating a sub domain on top of matrix

**Pull request closed without merging**